app/views/api/v0/classified_listings/index.json.jbuilder

```diff
@@ -0,0 +1,36 @@
+json.array! @classified_listings do |listing|
+  json.type_of              "listings"
+  json.id                   listing.id
+  json.title                listing.title
+  json.slug                 listing.slug
+  json.body_markdown        listing.body_markdown
+  json.bumped_at            listing.bumped_at
+  json.cached_tag_list      listing.cached_tag_list
+  json.category             listing.category
+  json.contact_via_connect  listing.contact_via_connect
```

Comment:
probably should stay private, this feels like an internal field

Reply:
Sorry again, I cannot figure which particular field we are talking here.

Reply:
`contact_via_connect`